### DISCUSSION TITLE
Processing broadcast data from SkillCorner

### DISCUSSION THREAD

**Discussion by ShirleyPmy** [PySport/kloppy#150] [Q&A] [answered]
Hi, I'm interested in the open data provided by SkillCorner. The format of tracking and event data transferred from the broadcast differs from Metrica or StatsBomb. Can kloppy deal with broadcast data? Or do you have any suggestions for processing broadcast data? Thanks a lot!

**Comment by probberechts** [comment, marked as answer]
I guess this is not relevant to you anymore, but for anyone else that would stumble on this: Yes, kloppy can parse SkillCorner data. You can find some example code here: https://kloppy.pysport.org/getting-started/skillcorner/
